Homework_7/Task_4.py

```diff
@@ -0,0 +1,11 @@
+elements = [1, 5, 68, 0]
+min_value = None
+
+for element in elements:
+    if min_value is None:
+        min_value = element
+    elif element < min_value:
+        min_value = element
+
+print(f"Min value in list is: {min_value}")
```

Comment:
Ca be better)
![image](https://github.com/vithaiduchyk/hillel_python_basic/assets/51053483/26b26397-bf61-4615-b12d-05c9d9531dba)

Reply:
I'll fix it